tests/topotests/lib/topogen.py

```diff
@@ -572,6 +573,10 @@ class TopoRouter(TopoGear):
         RD_BFD: "bfdd",
         RD_SHARP: "sharpd",
         RD_BABEL: "babeld",
+<<<<<<< HEAD
```

Comment:
Caught this running topotests but this `HEAD` shouldn't be here; artifact from an earlier rebase...